Spigot-Server-Patches/0725-Add-a-should-burn-in-sunlight-API-for-Phantoms-and-S.patch

```diff
@@ -0,0 +1,93 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MeFisto94 <MeFisto94@users.noreply.github.com>
+Date: Tue, 11 May 2021 00:48:33 +0200
+Subject: [PATCH] Add a "should burn in sunlight" API for Phantoms and
+ Skeletons
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/monster/EntityPhantom.java b/src/main/java/net/minecraft/world/entity/monster/EntityPhantom.java
+index 42cf3fa42b73739182d26fbb524ee5b304c799b2..779372084d4f7b7f13026e55e3fc81d331a57a6c 100644
+--- a/src/main/java/net/minecraft/world/entity/monster/EntityPhantom.java
++++ b/src/main/java/net/minecraft/world/entity/monster/EntityPhantom.java
+@@ -134,7 +134,7 @@ public class EntityPhantom extends EntityFlying implements IMonster {
+ 
+     @Override
+     public void movementTick() {
+-        if (this.isAlive() && this.eG()) {
++        if (this.isAlive() && burnInDay && this.eG()) { // Paper - Configurable Burning
```

Comment:
You should just be using isDrowning(), the patch you linked has the OBFHelper and is using the isDrowning() method in said patch. The mapping will most likely not get removed since it's an API patch and even if it did the helper would be moved somewhere else/retained.